src/Features/CSharp/Portable/GenerateType/CSharpGenerateTypeService.cs

```diff
@@ -945,7 +946,26 @@ internal override bool IsSimpleName(ExpressionSyntax expression)
                 newObjectCreation = (ObjectCreationExpressionSyntax)newNode.GetAnnotatedNodes(s_annotation).Single();
                 var symbolInfo = speculativeModel.GetSymbolInfo(newObjectCreation, cancellationToken);
                 var parameterTypes = newObjectCreation.ArgumentList.Arguments.Select(
-                    a => speculativeModel.GetTypeInfo(a.Expression, cancellationToken).ConvertedType).ToList();
+                    a =>
```

Comment:
I am not sure how to answer this question. Do you see any unnecessary parts?

---
In reply to: [68112768](https://github.com/dotnet/roslyn/pull/12159#discussion_r68112768) [](ancestors = 68112768)